### ISSUE TITLE
Unable to fetch all data from Apis that implement pagination mechanisms

### ISSUE THREAD

**Issue by dpsenner** [marcusolsson/grafana-json-datasource#302]
There are many api's out there that implement pagination mechanisms. To mention some:

1. Atlassian Jira
2. Teamviewer Api

It would be great if this plugin could support these apis better by fetching more data.

**Comment by vdufloth** [comment]
Hi @dpsenner where you able to connect to Jira using JSON API at all, or did this very issue prevented it from working?

With curl and postman I can fetch the data, but on Grafana it just says: "JSON API: Cannot connect to API"

![image](https://user-images.githubusercontent.com/43797735/231491590-621a0727-1894-4ff8-afba-99c846ee0216.png)

**Comment by dpsenner** [comment]
Yes, I could fetch the data but ended up with partial results. This is not at all useful and confusing at best.
